core/modules/system/js/system.admin.themes.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file system.admin.themes.js
+ *
+ * Behaviors for themes admin
+ */
+
+(function ($) {
+
+"use strict";
+
+/**
+ * Behavior for theme list
+ *
+ * Detect flexbox support for displaying our list of disabled themes
+ */
+Backdrop.behaviors.layoutDisplayEditor = {
+  attach: function(context) {
+    var $element = $(context).find('.system-themes-list-disabled');
+    if ($element.length) {
+      if ($element.css('flex-wrap')) {
+        $element.addClass('flexbox');
+      }
+      else {
+        $element.addClass('no-flexbox');
+      }
+    }
+  }
+};
+
+/**
+ * Behavior for showing a list of disabled themes.
+ *
+ * Detect flexbox support for displaying our list of themes with vertical
+ * height matching for each row of layout template icons.
+ */
+Backdrop.behaviors.disabledThemesListing = {
+  attach: function(context) {
+    var $element = $(context).find('.system-themes-list-disabled-wrapper');
```

Comment:
Couldn't this behavior be consolidated with the above behavior? They are identical except for the selector, so why not just select both of them and loop through them with `$('.selector').each()`?